aep/general/0008/aep.md.j2

```diff
@@ -215,6 +215,9 @@ API design examples in AEPs **should** be presented in both [OpenAPI][] and
 the concept. When using RPCs in examples, a `google.api.http` annotation
 **should** be included.
 
+[OpenAPI][] examples **must** be authored as
+[OAS version 3.1](https://spec.openapis.org/oas/v3.1.1.html).
+
```

Comment:
Do we want to go further? Should we require that all AEP tooling **must** support OAS version 3.1?

Reply:
I like that. Easier to draw the hard line now and then revisit if it proves too onerous.

Reply:
I think that makes sense. We shouldn't exclude aep tooling supporting older versions though (not that we could control that anyway).

Reply:
done! PTAL.